src/main/kotlin/skytils/skytilsmod/commands/stats/impl/CataCommand.kt

```diff
@@ -222,17 +222,13 @@ object CataCommand : StatCommand("skytilscata") {
         }
     }
 
-    private fun Duration.timeFormat() = toComponents { minutes, seconds, nanoseconds ->
+        fun Duration.timeFormat() = toComponents { minutes, seconds, nanoseconds ->
         buildString {
             if (minutes > 0) {
                 append(minutes)
                 append(':')
             }
             append("%02d".format(seconds))
-            if (nanoseconds != 0) {
```

Comment:
What happened here

Reply:
Dont accept that file change since a function will be added to ChatListeners.kt